perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_resource_attribute_def_def_unixGroupName_namespace.java

```diff
@@ -169,6 +171,8 @@ public void changedAttributeHook(PerunSessionImpl session, Resource resource, At
 	@Override
 	public List<String> getDependencies() {
 		List<String> dependencies = new ArrayList<String>();
+		dependencies.add(A_R_unixGID_namespace + ":*");
+		dependencies.add(A_G_unixGID_namespace + ":*");
 		dependencies.add(A_F_unixGroupName_namespace);
```

Comment:
Should `//dependencies.add(A_G_unixGroupName_namespace + ":*");` also really be there? Attribute is accessed but its value is not used.

Reply:
You are right. The best solution for now is to let it there commented, but also add information that we don't want to use this attribute in dependencies because it does not affect value of dependent attribute.